test/controllers/BankDetailsControllerSpec.scala

```diff
@@ -399,16 +460,18 @@ class BankDetailsControllerSpec extends ItSpec {
 
       val validForm: List[(String, String)] = formData
 
-      val expectedContentAndHref: List[(String, String)] =
+      val (expectedContentAndHref, expectedAuditResponseJson): (List[(String, String)], String) =
         // TODO temporary until error handling decided - will be addressed in a future ticket
         barsError match {
           case "accountNumberNotWellFormatted" =>
             BarsStub.ValidateStub.accountNumberNotWellFormatted()
-            List(("Enter a valid combination of bank account number and sort code", "#bars"))
+            List(("Enter a valid combination of bank account number and sort code", "#bars")) ->
+              ValidateJson.accountNumberNotWellFormatted
 
           case "sortCodeNotPresentOnEiscd" =>
             BarsStub.ValidateStub.sortCodeNotPresentOnEiscd()
-            List(("Enter a valid combination of bank account number and sort code", "#bars"))
+            List(("Enter a valid combination of bank account number and sort code", "#bars")) ->
+              ValidateJson.accountNumberNotWellFormatted
```

Comment:
and here. Something's not right I think

Reply:
corrected [here](https://github.com/hmrc/essttp-frontend/pull/107/commits/175f2805bac302d37abbab5bb5c0f2684eb2904c)